.github/workflows/changed_file.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--fileNameList', type=str, default='using_default_location')
+    args = parser.parse_args()
+    name_list = args.fileNameList.split(":")
+    folder_need_check = []
+    for loc in name_list:
+        # Find only the sub-folder of 'example' folder
+        if loc.split("/")[0] == "examples":
```

Comment:
This can raise exception if loc is a file under the root directory like `setup.py`.

Reply:
Emm...maybe this is OK? Because the first sub-folder should be examples? Thanks!